api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -276,17 +276,12 @@ message ExternalProcessor {
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
 
-  // [#not-implemented-hide:]
   // Send body to the side stream server once it arrives without waiting for the header response from that server.
   // It only works for STREAMED body processing mode. For any other body processing modes, it is ignored.
```

Comment:
mode override is not well supported in this feature.   like body is sent before override response is back.

Can you also add this to API?

Reply:
The mode override will still be honored if it is sent with header response, and the ext_proc filter is still in HeadersCallback state: (https://source.corp.google.com/piper///depot/google3/third_party/envoy/src/source/extensions/filters/http/ext_proc/processor_state.h;rcl=676834183;l=65)
https://github.com/envoyproxy/envoy/blob/6da051b1f229235ef5956df18c2894198ce3a807/source/extensions/filters/http/ext_proc/ext_proc.cc#L1068

For example, when this flag is enabled, if the side stream server header response reaches Envoy before the downstream client data reaches Envoy. Then it is still honored. i.e, nothing changes there.

Reply:
I think your reply above just demonstrate my point that it is not fully support.

That specific requirement is replying timing and downstream data most likely  come before stream server header response, which breaks mode override

Reply:
done